### PR TITLE
Update docker base image to use latest node LTS.

### PR DESCRIPTION
**Pull request by easternbloc**
Make docker image use nodejs user.

Ensure docker image is patched on build.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,10 +1,17 @@
-FROM quay.io/ukhomeofficedigital/nodejs-base:v6.9.1
+FROM quay.io/ukhomeofficedigital/nodejs-base:v6.11.0
 
 RUN mkdir /public
+RUN yum clean all && \
+  yum update -y -q && \
+  yum clean all && \
+  rpm --rebuilddb
```
Comment (JoeChapman) [inline Dockerfile]: Doesn't this lot happen `on-build` in the base image?

Comment (easternbloc) [inline Dockerfile]: 👀

Comment (easternbloc) [inline Dockerfile]: umm nope but we should probably add the onbuild there...

https://github.com/UKHomeOffice/docker-nodejs-base/blob/master/Dockerfile

**Comment by lennym** [comment]
Superceded by #540 so will close.

**Comment by lennym** [comment]
Actually, does the other - non-version-update - stuff need to still be done, and so does this need rebasing and merging? @easternbloc?

**Comment by easternbloc** [comment]
this needs a buttery biscuit yeah.

**Comment by JoeChapman** [comment]
resolve conflicts @easternbloc

**Comment by easternbloc** [comment]
thanks for the advise there!

**Comment by JoeChapman** [comment]
glad I could help @easternbloc

**Pull request merged**